### PR TITLE
Change 0 to <1 if value is less than 0.9999

### PR DESCRIPTION
**Pull request by shreeyash07**
- Addresses 
- #449
- #450  

Depends on:  fix/redis-key-generator

## Changes

* Change 0 to <1 in bar label list and pass nonNormalizedValue in regional Breakdown chart
* Change 0 to <1 in progress bar and change value in tooltip

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] eslint issues
- [x] typescript issues
- [x] codegen errors
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments
- [x] conflict markers

### CODE REVIEW

**Review by priyesh777 — APPROVED** [review]

File: app/utils/common.ts [hunk 1]
```diff
@@ -187,6 +187,22 @@ export const colors: Record<string, string> = {
...
 };
 
+export const normalizedValue = (
+    value: number | null | undefined,
+    format: FormatType,
+) => {
+    if (isNotDefined(value)) {
+        return 0;
+    }
+    if (value < 0.999) {
```
Comment (subinasr) [inline app/utils/common.ts]:
```suggestion
    if (value < 1) {
```

**Review by priyesh777 — APPROVED** [review]

File: app/components/PercentageStats/index.tsx [hunk 2]
```diff
@@ -47,8 +49,10 @@ function PercentageStats(props: Props) {
         statValueLoading,
         indicatorMonth,
         uncertaintyRange,
+        format,
     } = props;
 
+    // FIXME: use isNotDefined
```
Comment (subinasr) [inline app/components/PercentageStats/index.tsx]: Let's fix this!

File: app/components/CustomTooltip/index.tsx [hunk 3]
```diff
@@ -57,6 +57,7 @@ function CustomTooltip(props: Props) {
 
     const calculatedTotal = useMemo(() => (
         customTooltipData?.reduce(
+            // FIXME: this might not be correct
             (acc, obj) => (acc + (obj?.contextIndicatorValue ?? 1)), 0,
```
Comment (subinasr) [inline app/components/CustomTooltip/index.tsx]: Why is the value 1 here?

**Review by subinasr — APPROVED** [review]

File: app/components/CustomTooltip/index.tsx (hunk 3 above)
Comment (subinasr) [reply app/components/CustomTooltip/index.tsx:61]: .

**Review by subinasr — CHANGES_REQUESTED** [review]

**Review by subinasr — COMMENTED** [review]
Please fix the comments.

**Pull request merged**